la.ogri.strongbox.yml

```diff
@@ -0,0 +1,43 @@
+app-id: la.ogri.strongbox
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+    - org.freedesktop.Sdk.Extension.openjdk11
+
+command: strongbox
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+        - /usr/lib/sdk/openjdk11/install.sh
+
+    - name: metainfo
+      buildsystem: simple
+      sources:
+        - type: file
+          path: la.ogri.strongbox.metainfo.xml
+      build-commands:
+        - install -D la.ogri.strongbox.metainfo.xml /app/share/metainfo/la.ogri.strongbox.metainfo.xml
+
+    - name: strongbox
+      buildsystem: simple
+      sources:
+        - type: file
+          url: https://github.com/ogri-la/strongbox/releases/download/7.0.0/strongbox-7.0.0-standalone.jar
+          sha256: c2c254599773a3d16a1676da107a12000f592d1ceddb624ffd4ef918d1fd849d
+          dest-filename: app.jar
+        - type: script
+          commands:
+            - /app/jre/bin/java -jar /app/bin/app.jar "$@"
+
+      build-commands:
+        - install -D -m755 autogen.sh /app/bin/strongbox
+        - install -D app.jar /app/bin/app.jar
+
+finish-args:
+    - "--socket=x11"
+    - "--share=network"
+    - "--device=dri"
+    - "--filesystem=host"
```

Comment:
Why do you need full host?

Reply:
Linux users may have their WoW installation anywhere, sometimes multiple different installations or different wine prefixes. Within each installation there are now multiple addon directories.

The app downloads addons to a specific WoW installation in a specific addon directory and unzips it in place.

App config and app data otherwise follow XDG